include/cppkafka/utils/consumer_dispatcher.h

```diff
@@ -315,7 +315,7 @@ using ConsumerDispatcher = BasicConsumerDispatcher<Consumer>;
 
 template <typename ConsumerType>
 BasicConsumerDispatcher<ConsumerType>::BasicConsumerDispatcher(ConsumerType& consumer)
-: consumer_(consumer) {
+: consumer_(consumer),running_(false) {
```

Comment:
This one is not an issue, everything that checks the value of `running_` is called by `run()` (or one of its callees) which sets it to true.